### DISCUSSION TITLE
are you able to edit GitHub Spark code after it has been generated?

### DISCUSSION THREAD

**Discussion by alchEmily** [community/community#142999] [Other Feature Feedback, Questions, & Ideas]
I'm interested in using GitHub Spark as a learning tool for a middle-school student; are you able to view/edit the generated code?

**Comment by ebndev** [comment]
Hi @emilyCringe, thanks for being a part of the GitHub Community!

Yes, you are able to see the code after the app is generated. You can learn more about GitHub Spark [here](https://githubnext.com/projects/github-spark) and you might be interested in the [Managed runtime environment](https://githubnext.com/projects/github-spark#managed-runtime-environment) section regarding your question.

Let us know what your first Spark will be in [this discussion](https://github.com/orgs/community/discussions/142199)!

**Reply by alchEmily** [reply]
perfect; thank you so much
